doc/sections/options.md

```diff
@@ -1,134 +1,134 @@
-(http://cl2-informatik.uibk.ac.at/mercurial.cgi/TPDB/raw-file/tip/xml/xtc.xsd).
-The `--termination <cmd>` flag specifies the termination-checking command to
-be used. The command is expected to behave as follows:
- - take the problem description (in `.xml` format) on its standard input,
- - output on its first line either `YES`, `NO` or `MAYBE`.
-
-As an example,  `echo MAYBE` is the simplest possible (valid)
-termination-check that one may use.
-
-As far as we know,
-[`SizeChangeTool`](https://github.com/Deducteam/SizeChangeTool) is the
-only termination checker compatible with all `lambdapi` features.
-It can be called in the following way.
-```bash
-lambdapi --termination "path/to/sct.native --no-color --stdin=xml" input_file.lp
-```
-
-If the file does not contain type-level rewriting,
-[`Wanda`](http://wandahot.sourceforge.net/) is also compatible.
-However, it does not offer the possibility to give an input in `stdin`.
-
-To generate the `.xml` file corresponding to some `lambdapi` file, one may use
-a dummy termination-checking command as follows.
-```bash
-lambdapi --termination "cat > output.trs; echo MAYBE" input_file.lp
-```
-
-#### Debugging flags
-
-The following flags may be useful for debugging:
- - `--debug <flags>` enables the debugging modes specified by every character of
-   `<flags>`. Details on available character flags are obtained using
-   the `--help` flag.
- - `--timeout <int>` gives up type-checking after the given number of seconds.
+##### Debugging flags
+
+ - `--debug FLAGS` enables the debugging modes specified by every character of
+   `FLAGS`. Details on available character flags are obtained using `--help`.
+
+ - `--timeout NUM` gives up type-checking after the given number of seconds.
    Note that the timeout is reset between each file, and that the parameter of
    the command is expected to be a natural number.
- - `--too-long <float>` gives a warning for each command (i.e., file item) taking
-   more than the given number of seconds to be checked. The given parameter is
-   expected to be a floating point number.
 
-#### Rewriting engine
-
-The following options can be used to modify the behaviour of the
-reduction engine,
- - `--write-trees` writes the decision trees used for rule filtering
-   to dot files; for each symbol `s`, a file `mod_path/mod_name.s.gv` is
+ - `--write-trees` writes the decision trees used for rewriting rule filtering
+   to dot files. For each symbol `s`, a file `mod_path/mod_name.s.gv` is
    generated. For more information on trees (and how to read them), see
    `src/tree_graphviz.ml`.
- - `--keep-rule-order` forces the rewriting engine to use in priority
-   the topmost rules.
+
+##### Confluence checking
+
+Lambdapi provides an option `--confluence CMD` to check the confluence of the
+rewriting system by calling an external prover with the command `CMD`. The
+given command receives [TRS](http://project-coco.uibk.ac.at/problems/trs.php)
```

Comment:
Isn't it the HRS format (http://project-coco.uibk.ac.at/problems/hrs.php) instead?

Reply:
@Rehan-MALAK : you checked the use of csiho recently. Could  you please check this paragraph?